### PR TITLE
[#208] Fix wrong replacing string in variable server

### PR DESCRIPTION
**Pull request by marcuswu0814**
According to https://github.com/yonaskolb/SwagGen/issues/208#issuecomment-564533316 suggestion, it's not work for me. 🙁 Not sure if I understand it wrong.

But I using another solution to fix it, please review it and if have any better solution feel free to let me know. Thanks. 😀

### CODE REVIEW

**Comment by yonaskolb** [comment]
Thanks @marcuswu0814 , your approach seems to work too.
Could you please add a changelog entry as well

**Comment by marcuswu0814** [comment]
Hi @yonaskolb, I added changelog and feel free to modifled wording. Thanks. 🙏

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Change Log
 ## Next Version
 
+### Fixed
+- Fixed `'??' has non-optional type` warning #207
+- Fixed wrong replacing string in variable server #208
```
Comment (yonaskolb) [inline CHANGELOG.md]:
```suggestion
- Fixed incorrect replacements in server variables #209
```

**Comment by yonaskolb** [comment]
Thanks @marcuswu0814! :)

**Pull request merged**